porter/interfaces.py

```diff
@@ -77,3 +77,16 @@ def retrieve_cfrags(self,
             "retrieval_results": retrieval_outcomes
         }  # list of RetrievalOutcome objects
         return response_data
+
+    @attach_schema(schema.CBDDecrypt)
+    def cbd_decrypt(
+        self,
+        threshold: int,
```

Comment:
so the decrypter needs to know the threshold for the ritual that's being used?

Reply:
Good question. It's kind of an outstanding issue at the moment - https://github.com/nucypher/nucypher/issues/3095. Currently, Bob assumes the threshold is `n // 2 + 1` for the simple ferveo variant, and `n` for the pre-computed ferveo variant - https://github.com/nucypher/nucypher/blob/development/nucypher/characters/lawful.py#L650. Simple is used by default.

Either way, it feels like it will be known/obtainable.